src/utils/openmc/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
There is no `__main__`, so there is no need for the shebang.  This applies to all the file here.